CoreWebServer/src/au/gov/asd/tac/constellation/webserver/transport/FileListener.java

```diff
@@ -280,8 +280,7 @@ public void close() {
     private static OutputStream outStream(final Path p, final String name) {
         final Path fqp = p.resolve(name);
         try {
-            final OutputStream out = new FileOutputStream(fqp.toFile());
```

Comment:
@algol60 i merged this in and realised it was a WebServer change sorry. I'll deal with the merge conflict.